system-test/storage.ts

```diff
@@ -1217,6 +1217,66 @@ describe('storage', () => {
     });
   });
 
+  describe('cors configuration', () => {
+    const corsEntry = [
+      {
+        maxAgeSeconds: 1600
+      },
+      {
+        maxAgeSeconds: 3600,
+        method: [
+          "GET",
```

Comment:
Can you update all `"`s to `'`s? This will make it compliant with our linter.

Reply:
Did a linter fix.